specification/network/resource-manager/Microsoft.Network/preview/2024-03-01/webapplicationfirewall.json

```diff
@@ -783,6 +783,14 @@
           "items": {
             "$ref": "#/definitions/ManagedRuleGroupOverride"
           }
+        },
+        "computedDisabledRuleGroups": {
+          "readOnly": true,
+          "description": "Stores the computed list of disabled rule groups after consuming manifest",
```

Comment:
This description is also used in public docs, are we sure we want to add the bit about manifest? That may be confusing for users.